src/theme.ts

```diff
@@ -143,5 +143,64 @@ export const theme = createTheme({
         },
       },
     },
+    MuiStep: {
+      defaultProps: {},
+      styleOverrides: {
+        root: ({ ownerState, theme }) => ({
+          padding: 0,
+          marginRight: !ownerState.last ? "16px" : 0,
+          "&.Mui-disabled": {
+            //TODO: broken b/c MUI does not add this className as described in MUI docs
```

Comment:
I take it you want to leave these comments in? Or, can you remove them?

Reply:
would like to leave this in for when this MUI bug gets fixed